terraform/etl/48-lambda-gov-notify-ingestion.tf

```
@@ -176,3 +178,14 @@ resource "aws_cloudwatch_event_target" "govnotify_housing_repairs_trigger_event_
   EOF
 }
 
+resource "aws_glue_crawler" "govnotify_housing_repairs_landing_to_raw" {
+  database_name = "${local.identifier_prefix}-landing-zone-database"
+  name          = "${local.short_identifier_prefix}GovNotify Housing Repairs Landing to Raw"
+  role          = aws_iam_role.housing_gov_notify_ingestion[0].name
```

I think you want the glue role for the crawler
```suggestion
  role          = data.aws_iam_role.glue_role.arn
```